plasmapy/particles/tests/test_ionization_collection.py

```diff
@@ -814,9 +814,7 @@ def test_equality(self, this, that):
             print(f"{that} kwargs:\n {self.dict_of_kwargs[that]}\n")
             self.instances[that].summarize()
             descriptor = "equal" if expect_equality else "unequal"
-            pytest.fail(
-                f"Cases {this} and {that} should be {descriptor} but " f"are not."
```

Comment:
Sometimes when `black` autoformats things, it leaves two strings on the same line, since they get implicitly concatenated.